.node-version

```diff
@@ -1 +1 @@
-4.4.4
+6.6.0
```

Comment:
I tried reverting to 4.4.0 and the 404 error does not occur, so this is either a long-standing issue surfaced by Node 6, or a Node 6 specific issue.